src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -12,8 +12,15 @@ Release History
 ++++++++++++++++++
 * Minor fixes.
 
+0.1.6+dev
```

Comment:
Please just put "unreleased" here and it will get set appropriately during the release process